cue/private/extensions/cue.bzl

```diff
@@ -14,7 +14,7 @@ visibility("//cue")
 def _cue_impl(ctx):
     download_tool(
         name = "cue_tool",
-        version = maximal_selected_version(ctx, "Helm"),
```

Comment:
Thank you for catching this. It's been almost a year since I first made this mistake.

This typo came in as part of b6e63078c18c9a1cfb6d31e7ac1dbe85c3b2ee5f by way of copying and pasting from my _rules_kustomize_ ruleset, where [I used this technique first](https://github.com/seh/rules_kustomize/commit/3480a74b66c81740bff30da7939eb055f280b26b#diff-766e257837025eb87dd0e190cb9ea59476b9e744207eaf419ff1bdabf80d4cefR15). That containing commit has the same pasted typo, using "Helm" instead of "kustomize" (https://github.com/seh/rules_kustomize/commit/3480a74b66c81740bff30da7939eb055f280b26b#r136489577).

Reply:
I was looking into https://github.com/seh/rules_cue/issues/11 when I found this outlier 😊